libs/entitlement/tools/common/src/test/java/org/elasticsearch/entitlement/tools/AccessibleJdkMethodsTests.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.entitlement.tools;
+
+import org.elasticsearch.core.Tuple;
+import org.elasticsearch.entitlement.tools.AccessibleJdkMethods.AccessibleMethod;
+import org.elasticsearch.entitlement.tools.AccessibleJdkMethods.ModuleClass;
+import org.elasticsearch.test.ESTestCase;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.Matchers.hasItem;
+
+public class AccessibleJdkMethodsTests extends ESTestCase {
+
+    public void testKnownJdkMethods() throws IOException {
+        Map<ModuleClass, List<AccessibleMethod>> methodsByClass = AccessibleJdkMethods.loadAccessibleMethods(Utils.DEFAULT_MODULE_PREDICATE)
+            .collect(Collectors.groupingBy(Tuple::v1, Collectors.mapping(Tuple::v2, Collectors.toList())));
```

Comment:
nit: This sort of code gets more readable if you statically import the `Collectors` methods. It becomes:

```
.collect(groupingBy(Tuple::v1, mapping(Tuple::v2, toList())));
```

Side topic: I know you didn't write this part, but `Tuple` is obsolete now that Java has records, so `loadAccessibleMethods` should be returning a stream of records instead of a stream of tuples. Then this would be:

```
.collect(groupingBy(Foo::moduleClass, mapping(Foo::accessibleMethod, toList())));
```

Incidentally, this also highlights a weird naming quirk in this code: that `getAccessibleMethods` does not return a stream of `AccessibleMethod` objects.

...but I digress...

Reply:
++ on not using Tuples
I hate the "v1, v2" bits, you never know what they are (and which is which)